src/components/Notification/Notification.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Toast, ToastHeader, ToastBody, Button } from 'reactstrap'
+import Icon from '../Icon/Icon'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+const propTypes = {
+  header: PropTypes.string,
+  content: PropTypes.string,
+  state: PropTypes.string,
+  dismissable: PropTypes.bool,
+  fix: PropTypes.string,
+  withIcon: PropTypes.bool
+}
+
+// custom style
+const NotificationStyle = {
+  right: 'auto',
+  bottom: 'auto',
+  top: 'auto',
+  left: 'auto'
+}
+
+const NotificationWithIcon = props => {
+  const { header, content, state, dismissable, fix, withIcon } = props
+  const isDismissable = dismissable ? 'dismissable' : null
+  const isIcon = withIcon ? 'with-icon' : null
+  const wrapperClass = classNames(
+    'notification',
+    fix,
+    isIcon,
+    state,
+    isDismissable
+  )
+  var iconClass = 'it-check-circle'
```

Comment:
can you isolate this into a separate function?

```js
function pickIcon(status){
  if(status === 'success'){
    return 'it-check-circle';
  }
  ...
}

// ... in the component

const icon = pickIcon(status);
```

Also, I would consider to have a way from the user to pass his own icon as well: what about considering `withIcon` as a `boolean` value and `string` value?
If `boolean` it just toggles the icon.
If `string` it is used as `icon` name (hence enable the icon too).